utils/expand_csv_array_columns/expand_csv_array_columns.py

```diff
@@ -0,0 +1,59 @@
+"""
+This script loads csv files exported by ASHA fusion
+and expand columns which contain array values to separate columns like below.
+
+Example:
+    This example converts `source.csv` containing `medicines` column
+    whose value is JSON-encoded array to `converted.csv`
+    in which `medicines` column is expanded to separate columns
+    like `medicines_0` and `medicines_1`.
+
+        $ python expand_csv_array_columns.py \
+          source.csv converted.csv --array-columns medicines diagnoses
+
+source.csv
+----
+id,medicines,diagnoses
+1,"[""medicine-a"",""medicine-b""]","[""diagnosis-a"",""diagnosis-b"",""diagnosis-c""]"
+2,"[""medicine-c""]",
+----
+
+converted.csv
+----
+id,medicines_0,medicines_1,diagnoses_0,diagnoses_1,diagnoses_2
+1,medicine-a,medicine-b,diagnosis-a,diagnosis-b,diagnosis-c
+2,medicine-c,,,,
+----
+"""
+
+import argparse
+import json
+
+import pandas as pd
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('src', type=argparse.FileType('r'))
```

Comment:
I would recommend add the following attribute to clarify the usage of this argument:
help='path to source file'

Reply:
👍